pkg/api/devops/v1alpha4/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2020 The KubeSphere Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha4 contains API Schema definitions for the devops v1alpha4 API group
+//+kubebuilder:object:generate=true
+//+groupName=devops.kubesphere.io
+package v1alpha4
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var (
+	// GroupVersion is group version used to register these objects
+	GroupVersion = schema.GroupVersion{Group: "devops.kubesphere.io", Version: "v1alpha4"}
```

Comment:
How about creating a constant of these strings?

Reply:
According to [Kubernetes apps API definition](https://github.com/kubernetes/api/blob/2491c6b734cf94a7efa5cb41a5a6c8ef2d4f8400/apps/v1beta1/register.go#L26-L30), I created only `GroupName` constant and kept `Version` string.

Reply:
Good example.